examples/benchmarks/main.cpp

```diff
@@ -33,28 +36,28 @@
 using namespace tiny_dnn;
 using namespace std;
 
-int main(int argc, char** argv) {
-  CNN_UNREFERENCED_PARAMETER(argc);
-  CNN_UNREFERENCED_PARAMETER(argv);
-  models::alexnet nn;
+int main(int argc, char **argv) {
```

Comment:
I personally prefer previous one, however it's true that operators on the right could disambiguate things like 
```cpp
int* a, b  // a and b are pointers to int
int *a, a  // a is a pointer to int, b not
```